pkg/security/secl/model/category.go

```diff
@@ -37,7 +37,7 @@ func GetEventTypeCategory(eventType eval.EventType) EventCategory {
 	switch eventType {
 	case "exec", "signal", "exit", "fork":
 		return ProcessCategory
-	case "bpf", "selinux", "mmap", "mprotect", "ptrace", "load_module", "unload_module":
+	case "bpf", "selinux", "mmap", "mprotect", "ptrace", "load_module", "unload_module", "bind":
```

Comment:
kernel or network ?

Reply:
Initially I was wondering and start with network, but it automatically adds the network context (src/dst addresses etc) which make no sense for bind; so I switch it back to kernel (but I'm still not sure, as the difference between kernel and process categories aren't clear for me :))